src/less/material.less

```diff
@@ -187,7 +187,7 @@ regular-table, :host {
     }
 
     td th span.pd-group-leaf, th span.pd-group-leaf {
-        margin-left: 12px;
+        margin-left: 16px;
```

Comment:
This is a fix for the issue where the `<span class="pd-row-header-icon">` elements were throwing off the alignment of the dir/file names, right? Nice work

Reply:
Yes - Perspective has no way to generate datasets with "folders" at the same level as "leaves", so this number was off a few pixels.